src/snakemake/notebook.py

```
@@ -240,10 +240,49 @@ def get_interpreter_exec(self):
+        else:
+            self._execute_cmd("marimo edit {fname:q}", fname=fname)
+
+
 def get_exec_class(language):
     exec_class = {
         "jupyter_python": PythonJupyterNotebook,
         "jupyter_r": RJupyterNotebook,
+        "marimo_py": MarimoNotebook,
     }.get(language, None)
     if exec_class is None:
         raise ValueError("Unsupported notebook: Expecting Jupyter Notebook (.ipynb).")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)